### PR TITLE
added todo form

### PR DESCRIPTION
**Pull request by silchyn**
[DEMO](https://silchyn.github.io/react_add-todo-form/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,61 +1,129 @@
...
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [selectedTitle, setTitle] = useState('');
+  const [showTitleError, setTitleError] = useState(false);
+  const [showUserError, setUserError] = useState(false);
+  const [selectedUser, setUser] = useState('-1');
```
Comment (Oleksandr-Pustovit) [inline src/App.tsx:12]: it is recommended to use same names for value and setValue.
```suggestion
  const [selectedTitle, setSelectedTitle] = useState('');
  const [showTitleError, setTitleError] = useState(false);
  const [showUserError, setUserError] = useState(false);
  const [selectedUser, setSelectedUser] = useState('-1');
```

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1 +1,26 @@
...
+import users from '../../api/users';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const currentUser = users.find(user => user.id === todo.userId);
+
+  return (
```
Comment (Oleksandr-Pustovit) [inline src/components/TodoInfo/TodoInfo.tsx:12]: You should have prepared information about user in root component and sent user as a parameter or as a part of `todo`.
[checklist (5)](https://github.com/mate-academy/react_add-todo-form/blob/master/newChecklist.md)

**Review by Oleksandr-Pustovit — CHANGES_REQUESTED** [review]
Good job!
Several fixes needed.

**Review by Oleksandr-Pustovit — APPROVED** [review]
Nice!

File: src/components/TodoInfo/TodoInfo.tsx [hunk 3]
```diff
@@ -1 +1,26 @@
...
+  users: User[],
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo, users }) => {
+  const currentUser = users.find(user => user.id === todo.userId);
+
+  return (
+    <article
+      data-id={todo.id}
+      className={`TodoInfo${todo.completed ? ' TodoInfo--completed' : ''}`}
```
Comment (ivan-kadykalo) [inline src/components/TodoInfo/TodoInfo.tsx:15]: Use **classnames** library

**Review by ivan-kadykalo — APPROVED** [review]
🔥

**Pull request closed without merging**